src/features/vote/styled/RevealPhase.styled.tsx

```diff
@@ -7,6 +7,8 @@ interface StyledProps {
 export const Wrapper = styled.div<StyledProps>`
   &.RequestPhase {
     padding: 1rem 4rem;
+    width: 100%;
+    overflow-y: scroll;
```

Comment:
I think `overflow-y: auto` might be better here as `scroll` will always show a sidebar while `auto` will only show if the content overflows